seqr/views/utils/project_context_utils.py

```diff
@@ -165,42 +165,46 @@ def add_project_tag_types(projects_by_guid, add_counts=False):
             ),
             'variantFunctionalTagTypes': VariantFunctionalData.FUNCTIONAL_DATA_TAG_TYPES,
         })
-        if add_counts:
```

Comment:
this function can get tag types for multiple projects, but it is only ever called with `add_counts` for a single project, and there are a lot of db optimizations that can be made if we can assume a single project. Therefore, I added a separate utility function that adds the counts that only works on a single project